src/modules/file/file.repository.ts

```diff
@@ -201,6 +204,17 @@ export class SequelizeFileRepository implements FileRepository {
     });
   }
 
+  async findAllByParentUuid(
```

Comment:
This should be paginated. Also, as we have not finished the migration to uuid on folders, parent uuid is not still reliable, let's use id for now